docs/releasenotes.md

```diff
@@ -1,9 +1,38 @@
 ---
 layout: page-with-side-nav
-title: Haal Centraal BRP bevragingen
+title: Haal Centraal BRP Personen Bevragen
 ---
 
-# Releasenotes Haal-Centraal-BRP-Bevragen
+# Releasenotes Haal-Centraal BRP Personen Bevragen
+
+## Versie 2.0.0
+
+- de raadpleeg- en zoeken-endpoints zijn nu samengevoegd in 1 endpoint
+- het request is gewijzigd van GET naar POST, waarbij alle parameters verplaatst zijn van de querystring in de url naar de requestbody
+- requestbody is polymorf gedefinieerd, waarbij in parameter 'type' bepaalt welke parameters gebruikt mogen en moeten worden
+- parameter 'fields' is verplicht (required)
```

Comment:
hier nog iets meer over fields zeggen? Is een array, alleen volledig pad, niet vragen om inOnderzoek enz.